Floofbot/Modules/Utilities.cs

```diff
@@ -255,5 +257,127 @@ public async Task ServerList()
                 Trash = true
             }, true);
         }
+        [Command("convert")]
+        [Alias("conv")]
+        [Summary("Converts units to other units, such as Celcius to Fahrenheit.")]
+
+        public async Task convert(string input)
+        {
+            Regex fahReg = new Regex(@"\d+(?=f)", RegexOptions.IgnoreCase);
+            Regex celReg = new Regex(@"\d+(?=c)", RegexOptions.IgnoreCase);
+            Regex miReg = new Regex(@"\d+(?=mi)", RegexOptions.IgnoreCase);
+            Regex kmReg = new Regex(@"\d+(?=km)", RegexOptions.IgnoreCase);
+            Regex kgReg = new Regex(@"\d+(?=kg)", RegexOptions.IgnoreCase);
+            Regex lbReg = new Regex(@"\d+(?=lbs)", RegexOptions.IgnoreCase);
+
+            if (fahReg.IsMatch(input))
+            {
+                double fahTmp = Convert.ToDouble(Regex.Match(input, @"-?\d+").Value);
+
+                Temperature fah = Temperature.FromDegreesFahrenheit(fahTmp);
+                double cel = Math.Round(fah.DegreesCelsius, 2, MidpointRounding.ToEven);
+
+                EmbedBuilder builder = new EmbedBuilder()
```

Comment:
[OPTIONAL]

This is good for a first iteration, however, might be good to take the embed builder separate, then append to the description, then remove the 'else' in the 'else if' so that multiple conversations can be made on the same command. eg 

At the top of the command
```
string embedDescription = "";
```

Then in each command:

```
EmbedDescription += "🌡 {fah} is equal to {cel}°C.\n";
```

then at the end after all the ifs 
```
                EmbedBuilder builder = new EmbedBuilder()
                {
                    Title = "Conversion",
                    Description = embedDescription,
                    Color = EMBED_COLOR
                };
```
I think this is a cleaner, more handy way to use this command

Reply:
I updated the code with the new variable, however I didn't remove the `else` because the code doesn't handle more than one parameter at this time. I shall work on it later, though.

Reply:
> however I didn't remove the `else` because the code doesn't handle more than one parameter at this time. I shall work on it later, though.
 Don't mind this, it works with the `[Remainder]`

Reply:
It works but it keeps only the first number inputted, I'll work on it later on to see where it comes from.
Probably something to do with the Regex'ex.